pyodide-build/pyodide_build/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.21.0.dev0"
```

Comment:
Could you update `setup.cfg` of pyodide-build to reference this version variable (and remove setup.cfg from `bump_version`)?

AFAIK something like this will work:

```py
# inside setup.cfg
version = attr: pyodide_build.__init__.__version__
```

Reply:
We could probably do something similar in `src/py/setup.cfg` using the version from `src/py/pyodide/__init__.py`.

Reply:
Lately, I have been using more and more,
```py
from importlib.metadata import version, PackageNotFoundError

try:
    __version__ = version("package-name")
except PackageNotFoundError:
    # package is not installed
    pass
```
in package `__init__`. https://github.com/pypa/setuptools_scm#retrieving-package-version-at-runtime  It means that one only gets the version if the package is installed, but people shouldn't use the package if it isn't installed anyway.

Reply:
This sounds like a good idea, maybe we can switch to using this in the future.